### DISCUSSION TITLE
Unexpected error when extracting to C:\Program Files

### DISCUSSION THREAD

**Discussion by kilamen** [ALTaleX531/TranslucentFlyouts#97] [Show and tell]
![Screenshot 2023-11-20 053004](https://github.com/ALTaleX531/TranslucentFlyouts/assets/67429942/90b9fb8b-3a29-42c7-bee6-eb5e4eec7302)

### First time trying to set this up but it seems there's a problem when I extract the _rar file._

Any ideas? Would gladly appreciate any help and some troubleshooting! :pray:

**Comment by e-t-l** [comment]
%ProgramFiles% is a protected folder and you may not be able to extract directly into it, at least if you're not running the extraction software as Administrator. Have you tried extracting the files somewhere else, like the Desktop, then copying it all into the Program Files directory? (this is all assuming you have admin access on your Windows account.)
